{{cookiecutter.project_name}}/.github/workflows/build.yaml

```
@@ -15,15 +15,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python 3.12
-        uses: actions/setup-python@v5
         with:
-          python-version: "3.12"
-          cache: "pip"
-          cache-dependency-path: "**/pyproject.toml"
-      - name: Install build dependencies
-        run: python -m pip install --upgrade pip wheel twine build
+          filter: blob:none
+          fetch-depth: 0
+      - name: Install the latest version of uv
```

```suggestion
      - name: Install uv
```

Unnecessary detail. The version is encoded just below. I would hope that it gets regular updates through dependabot or our sync :)